pkg/smartcontract/manifest/standard/comply.go

```diff
@@ -73,5 +60,34 @@ func Comply(m, st *manifest.Manifest) error {
 			}
 		}
 	}
+	for _, stm := range st.Optional {
+		if err := checkMethod(m, &stm, true); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func checkMethod(m *manifest.Manifest, expected *manifest.Method, allowMissing bool) error {
+	actual := m.ABI.GetMethod(expected.Name, len(expected.Parameters))
+	if actual == nil {
+		if allowMissing {
+			return nil
+		}
+		return fmt.Errorf("%w: '%s' with %d parameters", ErrMethodMissing,
+			expected.Name, len(expected.Parameters))
+	} else if expected.ReturnType != actual.ReturnType {
```

Comment:
Can we remove this `else`? The code above has only `return` statements. Just to prettify the code.